lib/matplotlib/backends/_backend_tk.py

```diff
@@ -486,6 +486,20 @@ def create_with_canvas(cls, canvas_class, figure, num):
                 canvas.draw_idle()
             return manager
 
+    @classmethod
+    def start_main_loop(cls):
+        managers = Gcf.get_all_fig_managers()
+        if managers:
+            first_manager = managers[0]
+            manager_class = type(first_manager)
```

Comment:
```suggestion
            manager_class = cls
```

Reply:
this is copy-pasted from the previous implementation; I do not feel comfortable changing that.